app/models/verification_document.rb

```diff
@@ -31,7 +31,15 @@ def add_header(pdf)
 
   def header
     output = "Name:                  #{benefit_application.display_name}\n"
-    output += "Date of Birth:        #{benefit_application.birthday}\n"
+    output += "Date of Birth:        #{birthday}\n"
     output
   end
+
+  def birthday
+    if benefit_application.primary_member.birthday.present?
+      benefit_application.primary_member.formatted_birthday
+    else
+      ""
+    end
```

Comment:
I see `formatted_birthday` is a method on the `member` model. It would be cool if that were modified to account for nil values of `birthday` so that this above could be a one-liner:

`benefit_application.primary_member.formatted_birthday.presence`

Reply:
I think it's a nice-to-have, not essential.

Reply:
yeah, we could define `formatted_birthday` to allow for nil values. Per @hartsick's comment, maybe we don't want that method in the model at all since it's more of a view concern